Userland/Libraries/LibGfx/GradientPainting.cpp

```diff
@@ -213,18 +213,27 @@ static auto create_conic_gradient(ReadonlySpan<ColorStop> color_stops, FloatPoin
     };
 }
 
-static auto create_radial_gradient(IntRect const& physical_rect, ReadonlySpan<ColorStop> color_stops, IntPoint center, IntSize size, Optional<float> repeat_length)
+static auto create_radial_gradient(IntRect const& physical_rect, ReadonlySpan<ColorStop> color_stops, IntPoint center, IntSize size, Optional<float> repeat_length, Optional<float> radians = {})
```

Comment:
I think I'd prefer if this took the angle in degrees (to be consistent with other gradient functions), and if the parameter was simply called `rotation_angle`.